client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -388,6 +407,10 @@
                     criteria.source.size = 0; // we only need to get total num of items
                     scope.loading = true;
                     scope.total = null;
+                    if (desks.changeDesk) {
+                        desks.changeDesk=false;
```

Comment:
Please run `grunt hint` locally, I think it will complain about the missing spaces around the operator (and possibly along other things as well), causing the build to fail.